2-python-fundamentals-i.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `2-python-fundamentals-i.ipynb` Cell 154 Line 3_
```python
# Uncomment the below lines to run, but beware that this code will execute forever unless stopped. 
# while True:
    # print("infinite loop")
```
This is a critical section to have!

I'd go a step even further and explicitly advice people to avoid while loops. For DS in particular they're rarely needed, prefer to do a for loop instead.